python_modules/libraries/dagster-dbt/dagster_dbt/core/resources_v2.py

```diff
@@ -472,6 +486,18 @@ def stream_raw_events(self) -> Iterator[DbtCliEventMessage]:
                 if is_error_message:
                     self._error_messages.append(str(event))
 
+                # Attempt to parse the columns metadata from the event message.
+                # If it exists, then save it as metadata for the dbt node in the manifest.
```

Comment:
Is the "in the manifest" part of this comment still accurate?

Reply:
editted.